qcengine/programs/psi4.py

```diff
@@ -24,15 +26,32 @@ class Config(ProgramExecutor.Config):
     def __init__(self, **kwargs):
         super().__init__(**{**self._defaults, **kwargs})
 
+    @staticmethod
+    def found(raise_error=False) -> bool:
```

Comment:
Ah, we are changing all of these. If `raise_error` is necessary we should refactor some of the into the base class and just require `_found()` method.